vignettes/ExampleOfUse.Rmd

```diff
@@ -9,70 +9,106 @@ vignette: >
+    1.1. Split the regions into bins and aggregate methylation data in those bins to get a MIRA signature for each region set, sample combination.  
+    1.2. Calculate MIRA score for each region set, sample combination based on the shape of the MIRA signature.  
+2. Based on preliminary results, pick a region size for each region set depending on what the MIRA signatures look like.  
+3. Run the full MIRA analysis with chosen region sizes.  
+    3.1. Split the regions into bins and aggregate methylation data in those bins to get a MIRA signature for each region set, sample combination.  
+    3.2. Calculate MIRA score for each region set, sample combination based on the shape of the MIRA signature.  
+    
+The below code shows step 3 of the process (the step after choosing region sizes that are appropriate for your data) although the same thing would be done for steps 1 and 2, just with fewer samples and guesses for the region sizes. How to choose these sizes is discussed more below.
 
 
-```{r,eval=FALSE}
-#change to eval=TRUE when done
+```{r, eval = TRUE, message = FALSE}
 library(MIRA)
-library(data.table) #fread, setkey 
-load("exampleAnnoDF2.RData")#object name: "exampleAnnoDF2", data.table
+library(data.table) #fread, setkey, 
+library(GenomicRanges) #GRanges, resize
+data(exampleAnnoDF2)#object name: "exampleAnnoDF2", data.table
 ```
-```{r,eval=FALSE}
-caseFiles=c("RRBS_cpgMethylation_EWS_T10.bed","RRBS_cpgMethylation_EWS_T11.bed","RRBS_cpgMethylation_EWS_T12.bed")
-controlFiles=c("RRBS_cpgMethylation_MSC_10.bed","RRBS_cpgMethylation_MSC_11.bed","RRBS_cpgMethylation_MSC_12.bed")
-RRBSFiles=c(caseFiles,controlFiles)
-regionSetFiles=c("sknmc_specific.bed","muscle-specifichg38.bed")
-BSDTList=lapply(X = RRBSFiles,FUN = BSreadBiSeq)
-regionSets=lapply(X=regionSetFiles,FUN=fread)
-regionSets=lapply(X = regionSets,FUN = function(x) setNames(x,c("chr","start","end")))
-regionSets=lapply(X = regionSets, FUN = dtToGr)
-regionSets=lapply(X = regionSets, FUN = resize, 10000)
-names(regionSets) <- c("Ewing_Specific","Muscle_Specific")
+
+```{r, eval = FALSE}
+# 12 Ewing samples: T1-T12
+caseFiles = paste0("RRBS_cpgMethylation_EWS_T", 1:12, ".bed") 
+controlFiles = c("RRBS_cpgMethylation_Hsmm_", 
+                 "RRBS_cpgMethylation_Hsmmt_", 
+                 "RRBS_cpgMethylation_Hsmmfshd_",
+                 "RRBS_cpgMethylation_Hsmmtubefshd_")
+# 12 muscle related samples, 3 of each type
+controlFiles=paste0(controlFiles, rep(1:3, each = 4), ".bed")
+RRBSFiles = c(caseFiles, controlFiles)
+regionSetFiles = c("sknmc_specific.bed", "muscle-specific.bed")
+BSDTList = lapply(X = RRBSFiles, FUN = BSreadBiSeq)
+regionSets = lapply(X = regionSetFiles, FUN = fread)
+regionSets = lapply(X = regionSets, 
+                    FUN = function(x) setNames(x, c("chr", "start", "end")))
+#converting list items from data.table to GRanges objects
+#strand is not relevant for our region sets
+regionSets = lapply(X = regionSets, FUN = 
```

Comment:
It looks odd to have the argument on a separate line from the parameter. The way it is in the previous assignment (lines 57-58) looks better.